template/composer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "zendframework/<project-name>",
+    "description": "<project-description>",
+    "homepage": "https://docs.zendframework.com/<project-name>/",
+    "type": "library",
+    "license": "BSD-3-Clause",
+    "keywords": [
+        "<individual keywords for current project>"
```

Comment:
Good question. The keywords are used at searching and filtering. Maybe is "zend" as mark / brand also interesting?!